lib/src/r_tree/non_leaf_node.dart

```diff
@@ -28,6 +28,9 @@ class NonLeafNode<E> extends Node<E> {
         throw ArgumentError('too many items');
       }
       _childNodes = initialChildNodes;
```

Comment:
What's the downside of calling `addChild` for each initial child node here? 

And do you know why `addChild` does not push the new child node to the `_childNodes` array?

Reply:
> What's the downside of calling addChild for each initial child node here?

Curious about this too; I could imagine there being a difference in performance.

> And do you know why addChild does not push the new child node to the _childNodes array?

It does in a roundabout way: `addChild` calls `super.addChild` which does `children.add`, and `children` is just `_childNodes` for a `NonLeafNode`.

Reply:
just for performance reasons, calling addChild would recompute the bounding box once per child instead of doing it once at the end with the updateBoundingRect call

Reply:
addChild just calls `include`, so it could replace the updateBoundingRect call.

Reply:
oh yeah you guys are right, it would be the same number of iterations, I'll switch it since calling addChild is sturdier :)